app/controllers/question.js

```diff
@@ -50,4 +49,52 @@ exports.getQuestionsWithPagination = async (req, res) => {
             }
         }).catch(() => res.status(401).json({success: false, error: 'Ocorreu um erro ao buscar as questões.'}))
     }
+}
+
+const validate = (body) => {
+    return ['professor_id', 'coordinator_id', 'subarea_id', 'statement', 'approved', 'studyMaterials', 'comment'].every( field => body[field])
```

Comment:
Essa forma de validar vai falhar se 'approved' for `false` ou houver algum outro valor falsy intencional. E studyMaterials é opcional se não me engano.